fix_add_group_expense4.py

```
@@ -0,0 +1,20 @@
+import re
+
+f = 'lib/features/group_expenses/presentation/pages/add_group_expense_page.dart'
+with open(f, 'r') as file:
```

Ruff (0.15.5)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)